src/gui/Shared/SpectrumAnalyser.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+#include "SpectrumAnalyserTask.h"
+#include "dsp/MultiToolProcessor.h"
+
+class SpectrumAnalyser : public juce::Component,
+                         public juce::Timer
+{
+public:
+    SpectrumAnalyser (const chowdsp::SpectrumPlotBase& eqPlot, std::pair<gui::SpectrumAnalyserTask&, gui::SpectrumAnalyserTask&> spectrumAnalyserTasks);
+    ~SpectrumAnalyser() override;
+    void paint (juce::Graphics& g) override;
+    void visibilityChanged() override;
+    void timerCallback() override;
+    void updatePlotPath (juce::Path& pathToUpdate, gui::SpectrumAnalyserTask::SpectrumAnalyserBackgroundTask& taskToUpdate);
+
+    bool showPreEQ = true;
+    bool showPostEQ = true;
```

Comment:
Maybe we should add setter functions for these, so that the tasks don't need to keep running when we're not showing those visualizers? We might need to be a little more clever with how we're handling the task running logic elsewhere (e.g. in `visibilityChanged()`).